app/workers/conflate_stops_with_osm_worker.rb

```diff
@@ -2,7 +2,8 @@ class ConflateStopsWithOsmWorker
   include Sidekiq::Worker
 
   def perform(stop_ids = [])
-    stops = Stop.where(id: stop_ids)
-    Stop.conflate_with_osm(stops)
+    stops = Stop.where(id: stop_ids).find_in_batches(batch_size: 1000) do |batch_of_stops|
```

Comment:
Useless assignment to variable - `stops`. Did you mean `stop_ids`?<br>Line is too long. [90/80]